### PR TITLE
README: drop the logo and update TOC

### PR DESCRIPTION
**Pull request by mythi**
Signed-off-by: Mikko Ylinen <mikko.ylinen@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# ![Intel Device Plugins for Kubernetes](.intel-logo.png) Device Plugins for Kubernetes
+# Intel Device Plugins for Kubernetes
```
Comment (kad) [inline README.md:1]: `Intel ®` ?

Comment (mythi) [reply README.md:1]: I thought about it based on what we discussed but this is more consistent with other the READMEs we have

Comment (kad) [reply README.md:1]: I'm referring to other repositories brandings, like https://github.com/intel/intel-graphics-compiler

Comment (mythi) [reply README.md]: `&reg;` added.

**Pull request merged**